test/Serialization/containers.jl

```diff
@@ -139,6 +139,11 @@
           @test original == loaded
         end
       end
+
+      original = Dict{Symbol, Int}()
+      test_save_load_roundtrip(path, original) do loaded
+        @test original == loaded
```

Comment:
If you want to test that loading correctly infers the type parameters, this test is not semantically enough since 
```julia
julia> Dict{Symbol,Int}() == Dict()
true
```

Reply:
Great to know. Thanks!